### ISSUE TITLE
remove support for python 3.5

### ISSUE THREAD

**Issue by martinjrobins** [pybamm-team/PyBaMM#629]
Python 3.5 support is no longer needed, so remove testing from travis, and update documentation to say we no longer support it

**Comment by rtimms** [comment]
could make scikit.fem a requirement if we wanted to. was only optional as it required 3.6+

**Comment by tinosulzer** [comment]
do we still want separate tests for integration tests and examples? (in travis)

**Comment by martinjrobins** [comment]
Yes, we still need to run integration tests and examples for the other python versions
